app/models/article.rb

```diff
@@ -211,6 +204,18 @@ class Article < ApplicationRecord
     boosted_additional_tags String, default: ""
   end
 
+  def self.active_help
+    stories = published
+      .cached_tagged_with("help")
+      .order(created_at: :desc)
+      .where("published_at > ? AND comments_count < ? AND score > ?", 12.hours.ago, 6, -4)
```

Comment:
You can use the newer AR query syntax using endless ranges here:

```ruby
.where(published_at: 12.hours.ago.., comments_count: ..5, score: -3..
```

(Note that the ranges are inclusive, so instead of `< 6` and `> -4` this will become `<= 5` and `>= -3`.

Reply:
Love it!